### DISCUSSION TITLE
Updated Wifi/Bluetooth card to Intel AX210. Any way to show these in the G-Helper updates page instead of MediaTek?

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#1862] [Q&A]
The title

**Comment by Kikimahe** [comment]
Hi,

It is not possible.

Ghelper only connects to the Asus server.
The updates are therefore the same as on the Asus website for your laptop.

**Comment by seerge** [comment]
@akni23 pretty much as @Kikimahe said. G-Helper just shows you a mirror of Asus driver website, for your specific model. It doesn't scan your actual hardware for any custom added devices (like wifi adapter) :) so if it's not part of the asus original package - it won't be there.

I'm sure you can just download drivers directly from Intel
